collective/js/fancytree/configure.zcml

```diff
@@ -9,13 +9,46 @@
     directory="resources"
     />
 
+  <browser:resourceDirectory
+    name="collective.js.fancytree.theme-vista"
+    directory="resources/theme-vista"
+    />
+
+  <browser:resourceDirectory
+    name="collective.js.fancytree.theme-lion"
+    directory="resources/theme-lion"
+    />
+
   <genericsetup:registerProfile
       name="default"
-      title="collective.js.fancytree"
+      title="Fancytree jQuery plugin"
       directory="profiles/default"
       description='Extension profile for collective.js.fancytree.'
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
-</configure>
+  <genericsetup:registerProfile
+      name="all"
+      title="Fancytree jQuery plugin with all extensions"
+      directory="profiles/all"
+      description='Extension profile for collective.js.fancytree.'
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
 
+  <genericsetup:registerProfile
+      name="theme-lion"
+      title="Lion theme for Fancytree"
```

Comment:
Hi @tdesvenain 

could you please prefix every profile title with "Fancytree" so it is all grouped together in the portal_setup drop down for selecting the profile name?

Gauthier